presto-main/src/main/java/com/facebook/presto/server/BasicQueryInfo.java

```diff
@@ -225,43 +232,50 @@ public String getQuery()
 
     @ThriftField(9)
     @JsonProperty
+    public Optional<String> getPreparedQuery()
```

Comment:
can you add this at the end instead (i.e. as field 15) so you don't change the thrift field numbers?

Reply:
Sorry for the delay on this.
I was thinking of keeping similar attributes together to increase the readability/presentation of the code would be better.

Reply:
That makes sense, but best practice with thrift is not to shift around field numbers so that you can maintain compatibility across different versions of servers and clients.